docs/content-onboarding/onboarding.md

```diff
@@ -129,13 +132,13 @@ If the welcome message doesn’t appear, run SSH’s verbose mode by replacing `
 
 ## Additional Steps
 * Install Miniconda \(based on Python 3\) by following the [Conda How-To](https://draive.com/link_dev/guide/01_Conda_Setup/) and install the most important dependencies.
-* [Create an avatar](http://avatarmaker.com/) for your GitLab account. Add it to your GitLab [profile](https://gitlab.com/-/profile).
+* [Create an avatar](http://avatarmaker.com/) for your GitHub account. Add it to your GitHub [profile](https://github.com/login?return_to=https%3A%2F%2Fgithub.com%2F-%2Fprofile).
```

Comment:
I think that link is not correct... an you check?

Reply:
I have updated the GitHub profile link to https://github.com/settings/profile 
👍